src/partials/messenger.ts

```diff
@@ -410,7 +410,7 @@ class ConversationController {
                 $rootScope.$apply(() => {
                     this.updateScrollJump();
                 });
-            }, 100, this), supportsPassive() ? {passive: true} : false);
+            }, 0, this), supportsPassive() ? {passive: true} : false);
```

Comment:
Oh, before we merge this, can you completely remove the `throttle` call? It only causes unnecessary overhead when called with `0` as parameter.

Reply:
Good point. But I had problems when I initially removed the whole throttle-call. Anyway it could have been a side effect of some other changes I did not commit (I tried different approaches for several hours and then I was just glad that it was working).

So I will try it without the throttle and report back.

Thanks for reviewing!

Reply:
Yep, unfortunately this bug is the result of a race condition, so since removing the throttle function removes a layer of dispatching, it *could* make a difference whether it's there or not. Maybe you can test again?

The long-term solution will be a rewrite of the chat view. But a fix/workaround for now is great.

Reply:
Yeah, I also see this only as a workaround. I guess the root of the problem is `scroll-glue="!ctrl.showScrollJump"` in `messenger.converstion.html`.  It's accessing a (for 100ms) cached value since the value of `showScrollJump` only changes with a 100ms delay (caused by the throttle). But since the `scroll-glue` has no throttle, it overwrites the scroll position before `showScrollJump` is set. 

Additionally the 10px buffer for setting `showScrollJump` to true makes it even harder to detach from the `scroll-glue`.

I successfully tested it without the throttle (see commit 7842e0a) and updated my demo.

Reply:
Great, thank you!